apps/e2e/ag-grid-storybook-e2e/cypress.config.ts

```diff
@@ -14,6 +14,7 @@ const cypressPkg = JSON.parse(
 );
 
 export default defineConfig({
+  defaultCommandTimeout: 6000,
```

Comment:
AG Grid tests kept timing out without this longer timeout. The default is 4 seconds and 5 seconds was still flaky. We can chat about this if needed.